test/e2e/data/infrastructure-docker/main/clusterclass-quick-start-runtimesdk.yaml

```diff
@@ -49,6 +49,43 @@ spec:
       generateExtension: generate-patches.k8s-upgrade-with-runtimesdk
       validateExtension: validate-topology.k8s-upgrade-with-runtimesdk
       discoverVariablesExtension: discover-variables.k8s-upgrade-with-runtimesdk
+  - name: failSwap-controlPlane
```

Comment:
At this point can we just get rid of the patches and include this in all the templates instead of trying to do both?

Reply:
I don't think so.

When using the clusterclass its using the templates in the file of clusterclass. There is no kubeletExtraArgs defined there.

Reply:
Bump

Reply:
I think Killian means we can add this to KubeadmControlPlaneTemplate and KubeadmMachineTemplate (in the same file) directly without a patch

Reply:
Sorry for the back and forth

Reply:
moved away from patches and using kubeletExtraArgs now :-)